src/docs/product/performance/trends.mdx

```diff
@@ -11,8 +11,6 @@ From the Performance homepage, you can find the Trends View by toggling the tab
 Trending transactions are calculated first by filtering out transactions with large [throughput](/product/performance/metrics/#throughput-total-tpm-tps) fluctuations, determining the baselines of the two halves of the duration, and calculating their percentage change. Regressions are any percentage changes greater than zero, while improvements are percentage changes less than zero.
 Sentry uses a [t-test](https://academic.oup.com/beheco/article/17/4/688/215960) to compare the before and after periods of the transaction and assign a confidence score.
```

Comment:
Should this also be removed because it references the confidence score for old trends?